hadoop-hdfs-project/hadoop-hdfs-client/src/main/proto/hdfs.proto

```diff
@@ -563,13 +563,34 @@ message SnapshottableDirectoryStatusProto {
   required bytes parent_fullpath = 4;
 }
 
+/**
+ * Status of a snapshot directory: besides the normal information for
+ * a directory status, also include snapshot ID, and
+ * the full path of the parent directory.
+ */
+message SnapshotStatusProto {
+  required HdfsFileStatusProto dirStatus = 1;
+
+  // Fields specific for snapshot directory
+  required uint32 snapshotID = 2;
+  required bytes parent_fullpath = 3;
```

Comment:
we should also dispay if a particular snapshot is deleted or not.

Reply:
I think the intent was to hide the information from the user whether a snapshot is deleted or not. This is a user cmd and displaying it won't server the purpose if we are supposed to hide it.